qa/tasks/cephfs/mount.py

```diff
@@ -184,13 +184,19 @@ def _setup_netns(self):
             for ns in netns_list:
                 ns_name = ns.split()[0]
                 args = ["sudo", "bash", "-c",
```

Comment:
Is there a reason this is not:

`["sudo", "ip", "netns", ...]`

?

Reply:
No, just copied it from the mount.py code. I will fix it.

Reply:
For this, let keep it the same with others just like this ?   Or should we switch to "["sudo", "ip", "netns", ...]" ?

Reply:
Yes, please fix it.

Reply:
Okay, will fix them.